tests/foreman/cli/test_ping.py

```diff
@@ -46,6 +46,12 @@ def test_positive_ping(self):
 
         status_count = 0
         ok_count = 0
+        # Exclude message from stdout for services candlepin_events and katello_events
+        result.stdout = [
+            exclude_message
+            for exclude_message in result.stdout
+            if "message" not in exclude_message
```

Comment:
I think it would be more obvious if you have used `line` as variable name:
```
result.stdout = [line for line in result.stdout if "message" not in line]
```

Reply:
Thanks, @mirekdlugosz for the quick review, these formatting changes were added by black, will update the variable here 👍